presto-docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -82,7 +82,8 @@ address of the service account.
 The sheet needs to be mapped to a Presto table name. Specify a table name
 (column A) and the sheet ID (column B) in the metadata sheet. To refer
 to a specific tab in the sheet, add the tab name after the sheet ID, separated
-with ``#``.
+with ``#``. If tab name is not provided, connector loads only 10,000 rows by default from
+the first tab in the sheet (default name of the first tab is ``Sheet1``).
```

Comment:
> default name of the first tab is ``Sheet1``

Does this document how GS connector _finds_ the first tab? (i.e. no matter you reorder your tabs in web UI, the one with name "Sheet1" is considered the first)

Or does it document how GS connector _presents_ the first tab?

Also, would be worth mentioning explicitly what happens if there is _no tab_ of that name.

Reply:
This documents the default behavior of the G-sheet. If users have not renamed the tab, the first tab is named `Sheet1` by default. Adding tab name in sheet id is optional, in case, users don't add the tab name, GS-Client would always point to the first tab in the sheet whether the user has renamed it or not.

Reply:
thanks for explanation
then I'd suggest removing ```(default name of the first tab is ``Sheet1``)```, as it's stating the obvious... which led me to thinking it's saying something else.
(Also, "Sheet1" my be lang-dependent.)